src/lib/routes/admin-api/context.ts

```diff
@@ -156,6 +169,9 @@ export class ContextController extends Controller {
             middleware: [
                 openApiService.validPath({
                     tags: ['Context'],
+                    summary: 'Delete an existing context field',
+                    description:
+                        'Endpoint that allows deletion of a custom context field. Does not validate that context field is not in use, but since context field configuration is stored in a json blob for the strategy, existing strategies are safe.',
```

Comment:
The explanation here is probably useful, but seems to ... internal to Unleash (end users don't need to know how we store things). What does this mean to the user? If you delete a context field that is in use, what happens? 

```suggestion
                        'Endpoint that allows deletion of a custom context field. Does not validate that context field is not in use, but since context field configuration is stored in a json blob for the strategy, existing strategies are safe.',
```